
syntax = "proto3";

service Lya {
    rpc sync (SyncRequest) returns (SyncResponse);
}

message SyncRequest {
    repeated string function_names = 1;
    repeated string file_names = 2;
}

message SyncResponse {
    repeated string localizations = 1;
}